specification/_types/common.ts

```diff
@@ -40,6 +40,7 @@ export type ActionIds = string // TODO: check if this should be an array of Acti
 export type Id = string
 export type Ids = Id | Id[]
 export type NodeId = string
+export type NodeIds = string | string[]
```

Comment:
nit, can you please change to this:
`export type NodeIds = NodeId | NodeId[]`

Reply:
Good catch! Fixed.